Userland/Libraries/LibJS/Bytecode/Pass/MergeBlocks.cpp

```diff
@@ -119,7 +119,7 @@ void MergeBlocks::perform(PassPipelineExecutable& executable)
                 if (entry == cfg.end())
                     continue;
                 auto const* successor = *entry->value.begin();
-                if (successor == successors.first() && last != successor) {
+                if (successor == successors.first()) {
```

Comment:
that's not a sanity check

Reply:
Oops, that slipped into the wrong commit